### PR TITLE
Added Assets Page

### PR DESCRIPTION
**Pull request by JarrodRizor**
I created a assets page to download the Joind.in logo. Details on updates can be found on Jira.
https://joindin.jira.com/browse/JOINDIN-513

### CODE REVIEW

**Comment by akrabat** [comment]
ok to test

**Comment by akrabat** [comment]
ok to test

**Comment by akrabat** [comment]
Please rebase out the `.idea` folder.

File: app/templates/layout.html.twig [hunk 1]
```diff
@@ -45,7 +45,6 @@
                     <li>
                         <a href="{{ urlFor('about') }}">About</a>
                     </li>
-
```
Comment (akrabat) [inline app/templates/layout.html.twig]: Don't need this change.

**Comment by akrabat** [comment]
Thank you for your work on this. We really appreciate it and I'm super happy to finally have an assets page :)

**Comment by JarrodRizor** [comment]
No problem! Sorry I didn't rebase this sooner :(. I had a pretty hectic weekend.

I'll check out what else needs done and start working on other fixes/features.

**Pull request merged**